scripts/install/etc/sysconfig/motr

```diff
@@ -68,16 +68,16 @@ MOTR_M0D_MAX_RPC_MSG_SIZE=524288
 # (for example, to 512) to avoid packets drops on high loads.
 # Note: MOTR_M0D_IOS_BUFFER_POOL_SIZE and MOTR_M0D_SNS_BUFFER_POOL_SIZE
 # parameters should be updated also accordingly after this one.
-#MOTR_M0D_MIN_RPC_RECVQ_LEN=16
+MOTR_M0D_MIN_RPC_RECVQ_LEN=32
```

Comment:
I think currently 16 is being set as default, can you try with value 16 here ?

Reply:
I can update the PR please ask someone to try in day time.